app/src/main/res/layout/activity_main.xml

```diff
@@ -26,6 +26,7 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
+                android:textColorLink="#430404"
```

Comment:
What is this for?

Reply:
Not needed. that was a mistake. I have reverted it in a more recent commit.